pset-functions.js

```diff
@@ -31,6 +31,40 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+//return the card score of either player that is less than or equal to 21
+//difference of 21 - card score
+
+let playerCardScore = 0
+let dealerCardScore = 0
+let result = 0
+
+function blackJack(playerCardScore,dealerCardScore){
+  if (playerCardScore > 21 && dealerCardScore > 21){
```

Comment:
Daniel, really great use of if/else blocks to catch every possible scenario (in proper sequence as well!). Your code is very clean, which is to say highly readable (great use of indentation). A few suggestions: no need to declare the variable in the global scope (lines 37-39), the function parameters (line 41) will essentially serve as the variable that stores the arguments you pass when you call the function (line 60 below).